src/components/forms/material-form/material-form.js

```diff
@@ -237,8 +237,8 @@ function MaterialForm({ material, id }) {
                 label='Застосування'
               >
                 {Object.values(purposeEnum).map((value) => (
-                  <option key={value} value={value}>
-                    {value}
+                  <option key={value.en} value={value.en}>
```

Comment:
Unreadable, what are value.en and value.ua?

Reply:
Got it and fixed.